lib/workers/repository/dependency-dashboard.ts

```diff
@@ -453,3 +458,95 @@ function getFooter(config: RenovateConfig): string {
 
   return footer;
 }
+
+export async function getDashboardMarkdownVulnerabilities(
+  config: RenovateConfig,
+  packageFiles: Record<string, PackageFile[]>
+): Promise<string> {
+  let result = '';
+
+  if (
+    is.nullOrUndefined(config.dependencyDashboardOSVVulnerabilitySummary) ||
+    config.dependencyDashboardOSVVulnerabilitySummary === 'none'
+  ) {
+    return result;
+  }
+
+  result += '## Vulnerabilities\n\n';
+
+  const vulnerabilityFetcher = await Vulnerabilities.create();
+  const vulnerabilities = await vulnerabilityFetcher.fetchVulnerabilities(
+    config,
+    packageFiles
+  );
```

Comment:
Will this do duplicate work if both features are enabled?

Reply:
Both will hit down the stack tough with different parameters ( repo vs pr scope ): https://github.com/renovatebot/renovate/blob/8b06cdc2cb38811e80280062552b85007ef65e95/lib/workers/repository/process/vulnerabilities.ts#L93

IMO adding caching here would make much sense as this will have a higher latency then directly requesting entries from the offline osv database.

Reply:
Yeah, that's correct.    
What I've been referring to as "pr scope" is that `osvVulnerabilityAlerts` will request the vulnerabilities based on aggregated changes in the PRs/Updates.    

In contrast to here where all dependencies are looked up.